assets/js/modules/analytics-4/components/audience-segmentation/settings/SettingsCardVisitorGroups/SetupSuccess/index.js

```diff
@@ -34,35 +27,11 @@ import SetupSuccessContent from './SetupSuccessContent';
 const SetupSuccessContentWithIntersectionObserver =
 	withIntersectionObserver( SetupSuccessContent );
 
-export const SETTINGS_VISITOR_GROUPS_SETUP_SUCCESS_NOTIFICATION =
-	'settings_visitor_groups_setup_success_notification';
+export const SHOW_SETTINGS_VISITOR_GROUPS_SUCCESS_NOTIFICATION =
+	'showSetupSuccess';
```

Comment:
I understand this is how it was done before, but I think we should not declare the constant in two different places to ensure there is only one source of truth.

We should instead import it from `assets/js/modules/analytics-4/components/audience-segmentation/settings/SettingsCardVisitorGroups/SetupSuccess/SetupSuccessContent.js` and export it here to ensure the imports elsewhere still work.